decoupled-menu/src/main-menu.island.tsx

```diff
@@ -0,0 +1,369 @@
+import styled from "styled-components";
+import {useWebComponentEvents} from "./hooks/useWebComponentEvents";
+import {createIslandWebComponent} from 'preact-island'
+import {useState, useEffect, useRef, useCallback} from 'preact/hooks';
+import {deserialize} from "./tools/deserialize";
+import {buildMenuTree, MenuContentItem} from "./tools/build-menu-tree";
+import {DRUPAL_DOMAIN} from './config/env'
+import OutsideClickHandler from "./components/outside-click-handler";
+import Caret from "./components/caret";
+import Hamburger from "./components/hamburger";
+import Close from "./components/close";
+
+const islandName = 'main-menu-island'
+
+const MenuWrapper = styled.div<{ open?: boolean }>`
+  display: ${props => props.open ? "block" : "none"};
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  margin-left: calc(50% - 50vw);
+  margin-right: calc(50% - 50vw);
+
+  @media (min-width: 992px) {
+    display: block;
+    position: relative;
+    height: 100%;
+    width: 100%;
+    margin: 0 auto;
+  }
+`
+
+const TopList = styled.ul`
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  list-style: none;
+  margin: 0;
+  background: #fff;
+  padding: 0;
+  font-size: 18px;
+  box-shadow: 0 10px 20px rgba(0,0,0,.15),0 6px 6px rgba(0,0,0,.2);
+  border-top: 1px solid #D5D5D4;
+
+  @media (min-width: 992px) {
+    display: flex;
+    background: transparent;
+    height: 100%;
+    width: 100%;
+    box-shadow: none;
+    border-top: none;
+`
+
+const MobileMenuButton = styled.button`
+  position: absolute;
+  top: -70px;
```

Comment:
I'd boost this to -80px. Seems to look better. 

<img width="372" alt="Screenshot 2024-04-11 at 1 52 39 PM" src="https://github.com/SU-SWS/c_sharp_subtheme/assets/8387457/e673be87-bbfc-423d-a6f8-269ef04cc0ca">

Reply:
That was something the designer specifically requested - they wanted the `Menu` label to align with the bottom of the `c-ShSARP` text.